Makefile

```diff
@@ -59,12 +59,16 @@ $(COMPONENT_TARGETS):
 	$(DOCKER_BUILDX) bake $(PROGRESS) $@
 
 _LOCAL_PKG_DIR := $(OUTPUT)/$(DISTRO)
+ifneq ($(wildcard $(_LOCAL_PKG_DIR)/linux_*),)
+	_LOCAL_PKG_DIR := $(wildcard $(_LOCAL_PKG_DIR)/linux_*)
+endif
```

Comment:
Why is this necessary?

Reply:
Because the packages are under this dir.

Reply:
Without this, the tests don't use the packages from bundles.

Reply:
Hm, but the tests are working now without this change... the commits match during the containerd test for example. So the packages are making it into the aptly / local repo and getting installed. I'm fine with this change, just hope it doesn't break something.

Reply:
Are you sure about that? The reason I noticed this is because I tried to force the `--init` case to fail by changing the init path to something nonsensical and the tests still passed.
Changing this allowed the test to fail.
It was also installing moby-engine 23 even though 24 was in bundles and as such not installing moby-tini.